assets/html/themeswap.js

```diff
@@ -5,6 +5,14 @@ function set_theme_from_local_storage() {
   // Get the user-picked theme from localStorage. May be `null`, which means the default
   // theme.
   var theme =  window.localStorage.getItem("documenter-theme");
+  // Check if the browser supports user color preference
+  var darkPreference = false;
+  if(window.matchMedia('(prefers-color-scheme: dark)').media !== 'not all') {
```

Comment:
Is it actually necessary to check this? I assume it's here for browsers that do not support that query? It _looks_ like that even if you give `matchMedia` a rubbish query, it will still return a sane object with `matches === false`. So it might be sufficient to just read `window.matchMedia('(prefers-color-scheme: dark)').matches`, and if that's `false`, assume we should show a light theme by default. But I might be wrong -- I am not at all familiar with this API.